src/status_im/subs/wallet/activities.cljs

```diff
@@ -115,13 +115,13 @@
                                              (comp :network-name first))
          address-activities     (->> (get activities current-viewing-account-address)
                                      (vals)
-                                     (sort :timestamp))]
+                                     (sort-by :timestamp))]
      (->> address-activities
           (keep #(process-activity-by-type chain-id->network-name %))
           (group-by (fn [{:keys [timestamp]}]
                       (datetime/timestamp->relative-short-date (* timestamp 1000))))
           (map (fn [[date activities]]
                  {:title     date
-                  :data      activities
+                  :data      (reverse activities)
```

Comment:
Can we skip the `reverse` if we provide `>` or `<` in the `sort-by` call?

Reply:
I tried that and a lot of other methods, but none of them worked.

Reply:
The following code uses the idiom `#(compare %2 %1)`, this is a common way to pass a comparator to reverse the results of `sort-by`. With that, we can remove the `reverse` call, the subtraction workaround and the extra `sort-by` at the end, seems worth it. Tests still pass :)

```clojure
(defn activities-for-current
  [[activities current-viewing-account-address network-details]]
  (let [chain-id->network-name (update-vals (group-by :chain-id network-details)
                                            (comp :network-name first))
        address-activities     (->> (get activities current-viewing-account-address)
                                    (vals)
                                    (sort-by :timestamp #(compare %2 %1)))]
    (->> address-activities
         (keep #(process-activity-by-type chain-id->network-name %))
         (group-by (fn [{:keys [timestamp]}]
                     (datetime/timestamp->relative-short-date (* timestamp 1000))))
         (map (fn [[date activities]]
                {:title     date
                 :data      activities
                 :timestamp (:timestamp (last activities))})))))
```

Reply:
I also tried this method :( but still not working.

Reply:
I tried `(sort-by :timestamp #(compare %2 %1))` but still the tests were failing 🤔

Reply:
Oh. Got it now. I will try again

Reply:
I checked this. There is still one issue with it. It's sorting activities inside each day, but the days are not sorted. `(sort-by (fn [{:keys [timestamp]}] (- timestamp)))` this line of code is still needed.

Reply:
In that case, it would be good then to update the test to capture this requirement and falsify the snippet I shared, because the tests were passing. Well, it was all very interesting, thanks @mmilad75

Reply:
@ilmotta I will do so. Thanks for checking man